### DISCUSSION TITLE
Can we deploy and use this tool offline?

### DISCUSSION THREAD

**Discussion by cooltaby** [smart-test-ti/SoloX#200] [Q&A]
We have observed that tool fails to work without internet connection? how we can setup to use it as offline?

**Comment by rafa0128** [comment]
Offline is not supported

**Reply by cooltaby** [reply]
@rafa0128  Thanks for the answer, could you please share more details, why it requires the internet connection, and where it connects? just wanted to be sure where our game data can be uploaded?

**Reply by rafa0128** [reply]
Because it's a web service, not a software, the test datas will only be saved on the executing PC machine

**Reply by cooltaby** [reply]
Thanks for the quick response, is it possible to run this web service locally to communicate with Solo x? so that we can use it offline? like running socketio using node on local pc? it may sounds noob question but your answer will really appreciate.

**Reply by rafa0128** [reply]
I don't understand what you mean， but solox cannot be used offline because it is a web service and is not a static file or software

**Reply by cooltaby** [reply]
1: Can we host solox on specific ip and use it from other computers? 
2: I tried witthout internet, then run command python -m solox,  it started on 127.0.0.1:50003   Does it means it was working offline?
